src/components/ProjectConfigurationModal/ProjectConfigurationModal.test.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+import React from 'react';
+import { shallow } from 'enzyme';
+import {
+  ProjectConfigurationModal,
+  initialState,
+} from './ProjectConfigurationModal';
+import FormField from '../FormField';
+
+describe('ProjectConfigurationModal component', () => {
+  let wrapper;
+  let instance;
+  let mockActions;
+
+  const project = {
+    id: 'a-project',
+    name: 'A project',
+    projectIcon: 'icon',
+  };
+
+  const shallowRender = (installActive: boolean = false) => {
```

Comment:
If we're not having `// @flow` in this file, do we need to have boolean specified here?